### ISSUE TITLE
[DataCap Application] <FogMeta Lab> - <ComStock>

### ISSUE THREAD

**Issue by hengdingy** [filecoin-project/filecoin-plus-large-datasets#1803]
### Data Owner Name

FogMeta Lab

### Data Owner Country/Region

China

### Data Owner Industry

Web3 / Crypto

### Website

https://fogmeta.com

### Social Media

```text
Twitter: https://twitter.com/FogMeta
GitHub: https://github.com/FogMeta
```

### Total amount of DataCap being requested

5PiB

### Weekly allocation of DataCap requested

1PiB

### On-chain address for first allocation

f1tpennou4k6abbymhlolbbinsgnbt7keeya5zg3i

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
FogMeta Lab's research spans multiple levels from system technology, infrastructure, and middleware to services and solutions, and involves future systems, network technology and business, distributed systems and management, information management, and interactive and innovative services. Based on the views on and practices in the industry, FogMeta also solves the problem of business complexity through operations optimization and other technologies.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
“ComStock is a highly granular, bottom-up model that uses multiple data sources, statistical sampling methods, and advanced building energy simulations to estimate the annual sub-hourly energy consumption of the commercial building stock across the United States.”

Source: https://registry.opendata.aws/nrel-pds-building-stock/
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### How do you plan to prepare the dataset

IPFS, lotus, graphsplit, others/custom tool

### If you answered "other/custom tool" in the previous question, enter the details here

```text
We'd also like to use the Swan Client tool (https://github.com/filswan/go-swan-client#Graphsplit) to prepare the dataset.
```

### Please share a sample of the data

```text
s3://nrel-pds-building-stock/comstock/(196.2TiB)
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Monthly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, Africa, North America, South America, Europe, Australia (continent), Antarctica

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), HTTP or FTP server, IPFS, Shipping hard drives

### How do you plan to choose storage providers

Slack, Partners, Others

### If you answered "Others" in the previous question, what is the tool or platform you plan to use

```text
We'd also like to use FilSwan platform (https://filswan.com/) to choose storage providers who meet our requirements.
```

### If you already have a list of storage providers to work with, fill out their names and provider IDs below

```text
The storage providers we'd like to work with are presented below. Some of them are from the FilSwan platform.
f01955033
f02029115
f03624
f010088
f02301
f08399
f02401
f01955030
f0187709
f01163272
f01402814
f01390330
f01225882
f0717969
f03223
f01395673
f01072221
f0143858
f01786736
f0836160
f032824
f01443744
f01871352
f01907556
f01955028
f01947280
f01946551
f02012951
f01970630
f0240185
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client, Others/custom tool

### If you answered "Others/custom tool" in the previous question, enter the details here

```text
Swan Client tool
https://github.com/filswan/go-swan-client
```

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by Sunnyiscoming** [comment]
s3://nrel-pds-building-stock/comstock/(196.2TiB)
Why you need 5 PB datacap?

**Comment by hengdingy** [comment]
> s3://nrel-pds-building-stock/comstock/(196.2TiB)
> Why you need 5 PB datacap?

@Sunnyiscoming Considering the retrieval success rate of current storage providers's situation, we will store the 13 copies to the Filecoin network. The buckets data size about 196.2TiB, every piece size will be 16GiB(required 32GiB datacap), so all data need 196*1024/16*32*13/1024=5096TiB datacap

**Comment by zcfil** [comment]
After reading history, I have some questions：

Can you show more data？
Is your data publicly available？

**Comment by hengdingy** [comment]
@zcfil yes, the data is completely public. and all dataset is here: s3://nrel-pds-building-stock/comstock/(196.2TiB)

**Comment by hengdingy** [comment]
yes, we need notary help sign this application

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
keep alive

**Comment by hengdingy** [comment]
keep alive

**Comment by Normalnoise** [comment]
checker:manualTrigger

**Comment by Normalnoise** [comment]
checker:manualTrigger

**Comment by Normalnoise** [comment]
keep live

**Comment by Sunnyiscoming** [comment]
Hello, @hengdingy per the https://github.com/filecoin-project/notary-governance/issues/922 for Open, Public Dataset applicants, please complete the following [Fil+ registration form](https://form.jotform.com/231862753015656) to identify yourself as the applicant and also please add the contact information of the SP entities you are working with to store copies of the data.

This information will be reviewed by Fil+ Governance team to confirm validity and then the application will be allowed to move forward for additional notary review.

**Comment by hengdingy** [comment]
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1999#issuecomment-1824684456

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/2286#issuecomment-1835787739
@Sunnyiscoming

**Comment by hengdingy** [comment]
check

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
> ## DataCap and CID Checker Report[1](#user-content-fn-1-3fd8eacbe67deaebc2437a174e4b7787)
> No application info found for this issue on https://filplus.d.interplanetary.one/clients.
> 
> ## Footnotes
> 1. To manually trigger this report, add a comment with text `checker:manualTrigger` [↩](#user-content-fnref-1-3fd8eacbe67deaebc2437a174e4b7787)

checker:manualTrigger

**Comment by Kirameta** [comment]
manualTrigger

**Comment by hengdingy** [comment]
manualTrigger

**Comment by hengdingy** [comment]
manualTrigger

**Comment by Normalnoise** [comment]
checker:manualTrigger
